addons/common/functions/fnc_headBugFix.sqf

```diff
@@ -10,21 +10,30 @@
 
 #include "script_component.hpp"
 private ["_pos","_dir","_anim"];
-if (player != vehicle player  || {(player getvariable ["ace_isUnconscious", false])}) exitWith {};
+
+_anim = animationState ACE_player;
+["HeadbugFixUsed", [profileName, (animationState ACE_player)]] call FUNC(serverEvent);
+["HeadbugFixUsed", [profileName, (animationState ACE_player)]] call FUNC(localEvent);
+
+if (ACE_player != vehicle ACE_player  || { !([ACE_player, objNull, ["isNotSitting"]] call FUNC(canInteractWith)) } ) exitWith {false};
+
+_pos = getposATL ACE_player;
+_dir = getDir ACE_player;
+
 titleCut ["", "BLACK"];
-_pos = getposATL player;
-_dir = getDir player;
-_anim = animationState player;
+
 // create invisible headbug fix vehicle
-_ACE_HeadbugFix = createVehicle ["ACE_Headbug_Fix", getposATL player, [], 0, "NONE"];
+_ACE_HeadbugFix = createVehicle ["ACE_Headbug_Fix", [0,0,10000], [], 0, "NONE"];
```

Comment:
I think this is rare enough, the traffic shouldn't be that bad.  Also, I'm not sure if this is an issue, but it's important the animation is reset on all clients and a local vehicle might?? not sync correctly.

Minor, but moving the player to [0,0,10000] could cause issues with mission triggers (eg blufor not present), what about just
`(getposATL ACE_player) vectorAdd [0,0,10000]`